node/src/tasks/msp_charge_fees.rs

```diff
@@ -0,0 +1,129 @@
+use anyhow::anyhow;
+use sc_tracing::tracing::*;
+use shc_actors_framework::event_bus::EventHandler;
+use shc_blockchain_service::commands::BlockchainServiceInterface;
+use shc_blockchain_service::events::NotifyPeriod;
+use shc_blockchain_service::types::Tip;
+use shc_common::types::MaxUsersToCharge;
+use shc_common::types::StorageProviderId;
+use sp_core::Get;
+use storage_hub_runtime::Balance;
+
+use crate::tasks::{FileStorageT, MspForestStorageHandlerT, StorageHubHandler};
+
+const LOG_TARGET: &str = "msp-charge-fees-task";
+const MIN_DEBT: Balance = 0;
```

Comment:
These constants and the growing number of configuration parameters for BSPs/MSPs makes it increasingly compelling to have a way to configure them through a configuration file (like a .yml/.toml or similar). If you feel like that's something you'd like to do, setup a task with @santikaplan.